mods.go

```diff
@@ -321,6 +324,34 @@ func (m *Mods) startCompletionCmd(content string) tea.Cmd {
 			if api.BaseURL != "" {
 				ccfg.BaseURL = api.BaseURL
 			}
+		case "ollama":
+			occfg = DefaultOllamaConfig()
+			if api.BaseURL != "" {
+				ccfg.BaseURL = api.BaseURL
+			}
+		case "anthropic":
+			if key == "" {
+				key = os.Getenv("ANTHROPIC_API_KEY")
+			}
+			if key == "" {
+				return modsError{
+					reason: fmt.Sprintf(
+						"%[1]s required; set environment variable %[1]s or update mods.yaml through --settings.",
+						m.Styles.InlineCode.Render("ANTHROPIC_API_KEY"),
+					),
+					err: newUserErrorf(
+						"You can grab one at %s",
+						m.Styles.Link.Render("https://console.anthropic.com/settings/keys"),
+					),
+				}
+			}
+			accfg = DefaultAnthropicConfig(key)
+			if api.BaseURL != "" {
+				ccfg.BaseURL = api.BaseURL
+			}
+			if api.APIKey != "" {
+				accfg.Version = AnthropicAPIVersion(api.APIKey)
+			}
```

Comment:
this seems odd...

Reply:
Yes. It should be `api.Version` instead of `api.APIKey`. I've updated the code.